packages/osd-opensearch-archiver/src/actions/rebuild_all.ts

```diff
@@ -44,7 +44,7 @@ import {
 } from '../lib';
 
 async function isDirectory(path: string): Promise<boolean> {
-  const stats: Stats = await fromNode((cb) => stat(path, cb));
+  const stats: Stats = await promisify(stat)(path);
```

Comment:
Should use `fs/promises`:

```js
import { stat } from 'fs/promises';
```
```suggestion
  const stats: Stats = await stat(path);
```

Reply:
Sorry, I was sticking to description #3662
I'll look into it

Reply:
Don't apologize. I should thank you for doing all of this!